<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="PBAB">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: cyan</string>
    </property>
    <property name="text">
     <string>A -&gt; B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBAC">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue</string>
    </property>
    <property name="text">
     <string>A -&gt; C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBBA">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,0)</string>
    </property>
    <property name="text">
     <string>B -&gt; A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBBC">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue</string>
    </property>
    <property name="text">
     <string>B -&gt; C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBCA">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,0)</string>
    </property>
    <property name="text">
     <string>C -&gt; A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBCB">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: cyan</string>
    </property>
    <property name="text">
     <string>C -&gt; B</string>
    </property>
   </widget>
   <widget class="QLabel" name="MinimumLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum amount 
 of moves: </string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="min_moves">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="current_mov">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100)
</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurrLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Current amount 
 of moves: </string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Game status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="WinnerLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time spent: </string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Minutes">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100,200,250)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Seconds">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 50, 50)</string>
    </property>
   </widget>
   <widget class="QLabel" name="MinutesLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>minutes.</string>
    </property>
   </widget>
   <widget class="QLabel" name="SecondsLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>seconds.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QuitButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,0,0)</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>QuitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>686</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
